IonDotnet.Tests/Integration/VectorBad.cs

```diff
@@ -26,6 +26,7 @@ public class VectorBad
             "clobWithNonAsciiCharacter.ion",
             "clobWithShortLiteralBlockCommentAtEnd.ion",
             "clobWithShortLiteralInlineCommentAtEnd.ion",
+            "clobWithLongLiteralInlineCommentAtEnd.ion",
```

Comment:
Should this be in alphabetical order?

Reply:
It's not a must, I just put all the clobs together; they are removed in [another pull request](https://github.com/Armanbqt/ion-dotnet/pull/10) (as the fixed there).